pkg/w_apps_static/w_apps/find_cat.c

```diff
@@ -113,7 +113,7 @@ int main(int argc, char *argv[])
 	setvbuf(stdout, outbuf, BUF_SIZE, _IOFBF);
 
 	buf = malloc(BUF_SIZE);
-	catf = fopen("/usr/local/petget/categories.dat", "r");
+	catf = fopen("./rootfs-skeleton/usr/local/petget/categories.dat", "r");
```

Comment:
AFAIK 0setup is used by both woof-CE and PPM (inside the running Puppy) - if this is correct, this change will break PPM

Reply:
Correct - it is.

Reply:
Updated to use system-wide first, and if fails, use the local rootfs-skeleton version. Please see the second commit of this PR.

Reply:
Maybe it would be better to use the woof-CE file first, then fall back to the system one? i.e. if you're running woof-CE on some old Puppy with a broken categories.dat

Reply:
Sure, please see third commit.